src/styles/components/Accordion.scss

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2022 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+@import "../common/colors";
+@import "../common/fonts";
+
+.clr-accordion {
+    &-panel {
+        .clr-accordion-header {
+            background: $primary-white;
+        }
+        &-open {
+            & .clr-accordion-header {
+                font-weight: $font-weight-medium;
+                background: $blue-50;
+            }
+            .clr-accordion-title {
+                color: $blue-600;
```

Comment:
I think we need to keep title color to black even if it is active. Please check

Reply:
Referred[ this DDS page](https://web.ddsproduct.com/31b3fd8b1/p/12eb4b-navigation/b/4720a4/i/10975348) which has selected title color as in PR.